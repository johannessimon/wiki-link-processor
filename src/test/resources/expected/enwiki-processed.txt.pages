A_A_B
A_B
A_B_C
C_D
