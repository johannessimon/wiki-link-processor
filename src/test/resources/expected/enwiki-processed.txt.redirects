A_A_B	A_B
A_B_C	A_B
