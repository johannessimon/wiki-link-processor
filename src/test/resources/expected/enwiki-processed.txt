B C	B_C	This is a link from A B to <head>B C</head>.
a b	A_B	This is another link to <head>a b</head> with a custom alternative link text.
